Simplified/NYPLBookRegistry.m

```diff
@@ -187,6 +187,9 @@ - (void)loadWithoutBroadcastingForAccount:(NSString *)account
 
 - (void)save
 {
+  if ([AccountsManager.sharedInstance currentAccount] == nil) {
+    return;
+  }
```

Comment:
I've got some concerns about whether this nil check should also happen in other similar places like when the MyBooks tab gets opened. But if the account is effectively guaranteed to be populated by then, I dont see a problem.